platform/linux-dpdk/pool/dpdk.c

```diff
@@ -595,16 +594,18 @@ void odp_buffer_free_multi(const odp_buffer_t buf[], int num)
 	}
 }
 
-void odp_pool_print(odp_pool_t pool_hdl)
+static void dpdk_pool_print(odp_pool_t pool_hdl)
 {
 	uint32_t pool_id = pool_handle_to_index(pool_hdl);
 	pool_entry_t *pool = get_pool_entry(pool_id);
+
 	rte_mempool_dump(stdout, pool->s.rte_mempool);
```

Comment:
The API documentation says the print needs to go to ODP log. Is there a way to correct this?